rpc_jobs/phobos.yml

```diff
@@ -0,0 +1,194 @@
+- job:
+    name: Phobos-Deploy
+    project-type: workflow
+    concurrent: false
+    properties:
+      - build-discarder:
+          num-to-keep: 60
+    parameters:
+      # See params.yml
+      - rpc_gating_params
+      - rpc_repo_params:
+          RPC_BRANCH: newton
+      - rpc_eng_ops_params:
+          RPC_ENG_OPS_BRANCH: master
+      - string:
+          name: RPC_MAAS_BRANCH
+          default: master
+          description: |
+            Branch of rpc-maas repo used for verify-maas in the verification
+            stages. This does not affect the version of rpc-maas that is
+            deployed. That is determined by the version required by rpco.
+      - bool:
+          name: DEPLOY
+          default: true
+          description: Execute a deploy? Disable for veryify only. Jira issues are created for failures only if deploy is enabled.
```

Comment:
s/veryify/verify/

Reply:
Fixed the typo, I think the defaults should reflect the best settings for running a deploy. This will be important when we eventually automate this. 

This job is not safe, and shouldn't be executed by anyone who doesn't know what they are doing..